driver/scsi_operation.c

```diff
@@ -629,13 +629,13 @@ WnbdPendOperation(_In_ PWNBD_EXTENSION DeviceExtension,
         }
         break;
     default:
-        WNBD_LOG_ERROR("Unknown Pending SCSI Operation received");
+        WNBD_LOG_INFO("Unknown Pending SCSI Operation received");
```

Comment:
Let's update the message to WNBD_LOG_INFO("Unknown SCSI operation: %#x", Cdb->AsByte[0])